unit-tests-java8/src/test/java/org/eclipse/collections/test/bag/mutable/sorted/MutableBagIterableTestCase.java

```diff
@@ -37,6 +38,10 @@ default void MutableBagIterable_addOccurrences()
         assertThrows(
                 IllegalArgumentException.class,
                 () -> mutableBag.addOccurrences(4, -1));
+        assertEquals(Bags.immutable.with(1, 1, 1, 2, 2, 3, 3, 3, 4, 4, 4, 4), mutableBag);
+
+        Assert.assertEquals(-2147483647, mutableBag.addOccurrences(3, Integer.MAX_VALUE - 1));
+        Assert.assertEquals(-2147483638, mutableBag.size());
```

Comment:
Here I added assertions to show that addOccurrences and size already can overflows.